CODEOWNERS

```diff
@@ -3,68 +3,52 @@
 
 # Only files explicitly mentioned are guarded by code owners (no * rule).
 
-CODEOWNERS   @gerolf-da @remyhaemmerle-da
-LATEST       @gerolf-da @bame-da
+CODEOWNERS   @basvangijzel-DA @dylant-da @gerolf-da @paulbrauner-da @remyhaemmerle-da 
+LATEST       @basvangijzel-DA @dylant-da @gerolf-da @paulbrauner-da @remyhaemmerle-da
```

Comment:
Similarly, can we create a new group, daml-release-owners for this?

Reply:
I imagine we have to ask @nycnewman for creating a @digital-asset/daml-release-owners group. 
@nycnewman can you do that ?

Reply:
I've done it. (See https://github.com/orgs/digital-asset/teams/daml-release-owners/members to manager and https://github.com/orgs/digital-asset/teams/ to create new teams.)